cmd/mesh/manifest-migrate.go

```diff
@@ -84,7 +84,7 @@ func migrateFromFiles(rootArgs *rootArgs, args []string, l *logger) {
 
 // translateFunc translates the input values and output the result
 func translateFunc(values []byte, l *logger) {
-	ts, err := translate.NewReverseTranslator(version.NewMinorVersion(1, 3))
+	ts, err := translate.NewReverseTranslator(version.NewMinorVersion(1, 4))
```

Comment:
This is one of the places where hard coding should be taken out and ./version package used.